### PR TITLE
feat: Enhance suggested commits and add to alerts

### PR DESCRIPTION
**Pull request by dcramer**
- Refactor components to use new shared CommitRow
- Add Suspect Commits to alert emails
- Refactor committers scanning code to handle various edge cases.
- Add score to commit response
- Hide right column (SHA) on xs layouts [not ideal]

### CODE REVIEW

**Comment by dcramer** [comment]
screen shot 2018-03-01 at 12 08 03 pm

**Comment by dcramer** [comment]
@ckj i lowered the font size on headers but thats about it

**Comment by ckj** [comment]
@dcramer 👍

File: src/sentry/api/endpoints/event_file_committers.py [hunk 1]
```diff
@@ -145,65 +32,28 @@ def get(self, request, project, event_id):
...
+        #         'frame': frame['frame'],
+        #         'commits': serialize(frame['commits'])
+        #     } for frame in annotated_frames
+        # ]
 
         data = {
-            # map author ids to sentry user dicts
             'committers': committers,
-            'annotatedFrames': serialized_annotated_frames
+            # 'annotatedFrames': serialized_annotated_frames
```
Comment (macqueen) [inline src/sentry/api/endpoints/event_file_committers.py]: should this and the above just be deleted?

**Review by macqueen — APPROVED** [review]

File: src/sentry/api/endpoints/event_file_committers.py (hunk 1 above)
Comment (dcramer) [reply src/sentry/api/endpoints/event_file_committers.py]: I only kept it in that I think we're going to want it and its easier to uncomment code thatn it is to dig into git history

**Pull request merged**